cli/aeon/cmd/connection.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/tt/cli/util"
+	libcluster "github.com/tarantool/tt/lib/cluster"
+	libconnect "github.com/tarantool/tt/lib/connect"
+)
+
+func ShowUri(connectCtx *ConnectCtx, uriOpts libconnect.UriOpts,
+	instanceName string, collectors libcluster.CollectorFactory) error {
```

Comment:
The method actually do not show a URI, but fills the context.
```suggestion
func fillConnectCtx(connectCtx *ConnectCtx, uriOpts libconnect.UriOpts,
	instanceName string, collectors libcluster.CollectorFactory) error {
```